src/reason-parser/reason_parser.mly

```diff
@@ -59,6 +59,12 @@ open Ast_helper
 open Ast_mapper
 open Reason_string
 
+module Clflags = struct
+  include Clflags
+
+  let fast = unsafe
```

Comment:
This is pretty annoying. It won't build in <4.07 so we might have to introduce yet another `.cppo` compat file